### PR TITLE
fix(i18n): download english trending for fallback

### PR DESCRIPTION
**Pull request by ojeytonwilliams**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Once we're 100% sure that the trending data will be available for all languages, we can remove the fallback, but for now this seems like the prudent approach.

### CODE REVIEW

**Comment by ojeytonwilliams** [comment]
@raisedadead this should fix the pipelines.

File: tools/scripts/build/download-trending.ts [hunk 1]
```diff
@@ -44,4 +44,7 @@ const download = async () => {
   }
 };
 
-void download();
+void download(clientLocale);
+// TODO: remove the need to fallback to english once we're confident it's
+// unnecessary (client/i18n/config.js will need all references to 'en' removing)
+void download('english');
```
Comment (scissorsneedfoodtoo) [inline tools/scripts/build/download-trending.ts:50]: Won't make much of a difference in the end, but maybe slightly more efficient for English builds:

```suggestion
if (clientLocale !== 'english') void download('english');
```

**Review by scissorsneedfoodtoo — APPROVED** [review]
Noticed one small thing, but not necessary at all.

Either way, everything LGTM.

**Comment by ojeytonwilliams** [comment]
Cheers, @scissorsneedfoodtoo thanks for fixing that!

**Review by ahmaxed — APPROVED** [review]

**Pull request merged**